README.md

```diff
@@ -1,6 +1,15 @@
 # Powertools for AWS Lambda (TypeScript) <!-- omit in toc -->
 
-[![Join our Discord](https://dcbadge.vercel.app/api/server/B8zZKbbyET)](https://discord.gg/B8zZKbbyET)
+[![Build](https://github.com/aws-powertools/powertools-lambda-typescript/actions/workflows/pr-run-linting-check-and-unit-tests.yml/badge.svg)]([https://github.com/aws-powertools/powertools-lambda-typescript/actions/workflows/pr-run-linting-check-and-unit-tests.yml])
+![NodeSupport](https://img.shields.io/static/v1?label=node&message=%2016|%2018|%2020&color=green?style=flat-square&logo=node)
+![GitHub Release](https://img.shields.io/github/v/release/aws-powertools/powertools-lambda-typescript?style=flat-square)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=aws-powertools_powertools-lambda-typescript&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=aws-powertools_powertools-lambda-typescript)
+[![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=aws-powertools_powertools-lambda-typescript&metric=security_rating)](https://sonarcloud.io/summary/new_code?id=aws-powertools_powertools-lambda-typescript)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/aws-powertools/powertools-lambda-typescript/badge)](https://api.securityscorecards.dev/projects/github.com/aws-powertools/powertools-lambda-typescript)
+![NPM Downloads](https://img.shields.io/npm/dw/%40aws-lambda-powertools%2Ftracer?style=flat-square&label=Tracer)
```

Comment:
I would remove these, we have 3 more packages and soon we'll have more. I think listing only the core ones reinforces the notion that we only do observability - which we are trying to get away from.

In the future we could use a custom badge that aggregates all of them, but overall I think it's ok not to show this for now.

Reply:
We initially discussed the option of adding all packages, but it would be too much information overload. I agree with you on an unintentional signal for "observability only", and change my mind to remove the badges for downloads for now.